src/C1T2X_OBU.py

```diff
@@ -79,7 +79,7 @@
 	c1t2x_logger.warning("Configured LOGGING LEVEL is invalid. Level is set to WARNING.")
 
 
-# instantiate networks
+### Instantiate networks
 # LAN
 try:
```

Comment:
`try` should `except` a specific error.

> `raise ImportError`

Why is this raising an `ImportError`? It should be changed to a more appropriate one.